lib/metrics/prometheus.js

```diff
@@ -47,49 +49,56 @@ class PrometheusMetric {
     }
 
     increment(amount, labels) {
-        this._handleStaticLabels(labels);
-        labels = labels.map(this._normalize);
-        this.metric.labels.apply(this.metric, labels).inc(amount);
+        let updatedLabels = labels;
```

Comment:
Javascript doesn't create a copy on assignment. This is still pointing to exact same array. This patch changes nothing...

You need to actually clone the labels array... let updatedLabels = [...labels] will work. Just check this syntax works with node 6. Alternatively, let updatedLabels = labels.map(label => label) should work too.